index.html

```diff
@@ -413,12 +405,42 @@ <h2><code>PointerEvent</code> Interface</h2>
                            </ol>
                            </p>
                         </dd>
+                      <dt><dfn>getAzimuthAngle</dfn> <code>(long tiltX, long tiltY)</code></dt>
+                        <dd>
+                            Return the <code>azimuthAngle</code> angle (in radians) of the transducer (e.g. pen stylus) calculated from
+                            <code>tiltX</code> and <code>tiltY</code> angles, in the range [0, 2π] - where 0 represents a transducer whose cap is pointing in the
+                            direction of increasing X values (point to "3 o'clock" if looking straight down) on the X-Y plane, and the
+                            values progressively increase when going clockwise (π/2 at "6 o'clock", π at "9 o'clock", 3π/2 at "12 o'clock").
+                            When the transducer is perfectly perpendicular to the surface (<code>tiltX</code> and <code>tiltY</code> of 0),
+                            the returned value MUST be 0.
+                        </dd>
```

Comment:
It's not clear to me as an implementer how I would implement this method. In particular this section describes the return value, but does not give an algorithm for computing it. The only normative requirements is that `getAzimuthAngle(0, 0)` must return `0`, but it says nothing about the other 2^32 values for each of tiltX and tiltY.

Same for the other three methods.

Reply:
There is a "Converting between `tiltX`/`tiltY` and `altitudeAngle`/`azimuthAngle`" section which provides Javascript code for conversion back and forth. Is that enough?
For all methods I need to provide direction on handling inputs outside of the supported intervals: `[-90,90]` for `tiltX`, `tiltY` and respectively: `[0,π/2]` for `altitudeAngle` and `[0,2π]` for `azimuthAngle`. Would it be ok to recommend the methods throw RangeError exceptions for invalid input?